torch/_subclasses/meta_utils.py

```diff
@@ -845,6 +845,14 @@ def _empty_create_subclass(
                     )
                     inner_tensors[attr] = new_empty_tensor
 
+                if t.is_nested:
+                    # Invariant: A fake tensor post-creation is guaranteed to
+                    # have any associated symbolic nested int set
```

Comment:
comment nit but should this be "An newly fakeified NJT is guaranteed to have its symbolic nested int set" (i.e. this logic is `.is_nested`-specific today, and NJT's always have one associated nested int)

Reply:
> An newly fakeified NJT is guaranteed to have its symbolic nested int se

I actually think I want to say something slightly stronger because suppose I have:
```
fn(values, offsets):
   ...
   return = NJT(values, offsets)
   ...
values = ...
offsets = ...
nt = NJT(values, offsets)  # associate offsets with nested int in eager
nt1 = compiled_fn(values, offsets)
nt * nt1
```
My function doesn't actually take NJT as inputs. I know however, that when the offsets is fakified (guaranteed to happen before the NJT in the graph is constructed) I can check the eager global state to figure out if it has been associated with a nested int, and cache it, so that when NJT gets constructed in the graph later, I won't use a fresh id/increment the counter. This is important in order for a check later like nt * nt1 to succeed.

Reply:
hmm this makes sense - I was confued because the comment lives inside a region of code guarded with `t.is_nested` (so this code won't actually run unless we are fakifying a NJT directly, right?)